crates/core/tedge_actors/src/message_boxes.rs

```diff
@@ -209,12 +219,90 @@ impl<Input: Message, Output: Message> SimpleMessageBox<Input, Output> {
     }
 }
 
+#[async_trait]
+impl<Input: Send + Debug, Output> ReceiveMessages<Input> for SimpleMessageBox<Input, Output> {
+    async fn try_recv(&mut self) -> Result<Option<Input>, RuntimeRequest> {
+        self.receiver.try_recv().await
+    }
+
+    async fn recv_message(&mut self) -> Option<WrappedInput<Input>> {
+        self.receiver.recv_message().await
+    }
+
+    async fn recv(&mut self) -> Option<Input> {
+        self.receiver.recv().await
+    }
+}
+
+pub struct CombinedReceiver<Input> {
+    name: String,
+    logging_is_on: bool,
```

Comment:
For logging, we need to go away from this `logging_is_on` stuff. This has been done in a hurry - my bad.

With this new trait `ReceiveMessage` it will nicer to have `LoggingReceiver` that owns an inner receiver and logs the messages.